ui.frontend/src/view/FormContainer.js

```diff
@@ -69,6 +69,7 @@ export default class FormContainer {
             this._fields[fieldId] = fieldView;
             let model = this.getModel(fieldId);
             fieldView.setModel(model);
+            fieldView.syncMarkupWithModel();
```

Comment:
setModel and sync markup should be a unit operation. @nit23uec I still believe it should be invoked inside setModel of View.

Reply:
setModel's responsibility is only to set the model of the view and to modify the model on view changes (i.e. attaching event listeners).
pls note that it's not the container rather the fieldView that's syncing markup.